### PR TITLE
scripts and functions for standard brain

### PR DESCRIPTION
**Pull request by changsiyao**
changed saving path in findoutlier script
changed regression script to calculate t vals, save beta and t values.
add regression functions and scripts for standard data. (We don't need linear and quadratic drift for standard data.)
add plot heatmap script for plotting heatmaps for beta and t values. 
Jiang co-wrote the plotting heatmap script.

Jiang might have updated the scripts, since he has been running all the script on standard data. So please check, @pigriver123

### CODE REVIEW

**Comment by changsiyao** [comment]
Oh, I added a new function script for standard data, but the test should be similar. I don't know if you want to add another test, or if we want to somehow combine the two cases... Let me know your idea, @BenjaminHsieh

**Comment by briankleinqiu** [comment]
Can you upload the final heatmaps image?

Also how long does it take now to generate all the t-values?

**Comment by BenjaminHsieh** [comment]
ok let me take a look, I will probabily combine the two cases, will make a pull request later today

**Comment by changsiyao** [comment]
I only have the heatmaps for the original data. Jiang has the heatmaps for the preprocessed data.
Also, do you want all heatmaps? That's 32 heatmaps for beta values and 32 heatmaps for the t values.

**Comment by BenjaminHsieh** [comment]
Also missing calc_t module @pigriver123

File: code/scripts/regression_script_standard.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        run = j
+        behav_cond = 'ds005/sub0'+str(i).zfill(2)+'/behav/task001_run00'+`j`+'/behavdata.txt'
+        task_cond1 = 'ds005/sub0'+str(i).zfill(2)+'/model/model001/onsets/task001_run00'+`j`+'/cond001.txt'
+        task_cond2 = 'ds005/sub0'+str(i).zfill(2)+'/model/model001/onsets/task001_run00'+`j`+'/cond002.txt'
+        task_cond3 = 'ds005/sub0'+str(i).zfill(2)+'/model/model001/onsets/task001_run00'+`j`+'/cond003.txt'
+        task_cond4 = 'ds005/sub0'+str(i).zfill(2)+'/model/model001/onsets/task001_run00'+`j`+'/cond004.txt'
+        parameters = merge_cond(behav_cond, task_cond1, task_cond2, task_cond3, task_cond4)
+        neural_prediction = events2neural_extend(parameters,TR, n_vols)
+        gain, loss = getRegressor(TR, n_vols, hrf_at_trs, neural_prediction, standard = True)
+        data, gain, loss = deleteOutliers(data, gain, loss, i, run, dvars_out, fd_out)
```
Comment (BenjaminHsieh) [inline code/scripts/regression_script_standard.py]: For lines 49 -50, still need to assign a `lin` and `quad` as dummy variables in results (i think right now you will get an error if you run this script)

**Comment by BenjaminHsieh** [comment]
Ok yeah should be good (although I havent run the script yet)

**Comment by briankleinqiu** [comment]
Not sure how jiang is approaching the standard heatmaps but heres a thought:

at the last for-loop of regressions_script_standard we can check if the t-value is significant. If it is we then find apply the affine to that voxel to find the mm, and then save the mm coordinates of all significant voxels for the subject. 

Then we can overlay a plot of that on the standard brain template

**Pull request merged**